src/tools.py

```diff
@@ -713,6 +713,20 @@ def tools_basic_space_cleanup() -> None:
     subprocess.run("rm /var/log/*/*.gz", shell=True)
     subprocess.run("rm /var/log/*.?", shell=True)
     subprocess.run("rm /var/log/*/*.?", shell=True)
+def tools_clean_old_kernels() -> None:
+    # Removing kernel except last 2
+    uname_output = subprocess.run("uname -r", capture_output=True, text=True, shell=True)
+    # running_kernel is expected to be something like : 6.1.0-28-amd64
+    running_kernel = uname_output.stdout.strip()
+    dpkg_output = subprocess.run("dpkg -l | grep '^ii\s*linux-image' | awk '{print $2}'", capture_output=True, text=True, shell=True)
```

Comment:
```suggestion
    dpkg_output = subprocess.run("dpkg -l | grep '^ii\s*linux-image-[0-9]' | awk '{print $2}'", capture_output=True, text=True, shell=True)
```

Let's further refine the grep regex to only catch the "actual version" and not the `linux-image-amd64` (on other arch there may be other stuff like i have `linux-image-rpi-<stuff>` on my RPi